lib/src/state_manager.cc

```diff
@@ -61,6 +61,7 @@
 */
 
 #include "state_manager.h"
+//#include <fty_log.h>
```

Comment:
why not fully removing / leaving the below logs?

Reply:
done